pages/common/z.md

```diff
@@ -1,6 +1,6 @@
 # z
 
-> Tracks the most used directories and enables quickly navigating to them using string or regex patterns.
+> Tracks the most used (_frecent_, read frequently recent) directories and enables quickly navigating to them using string or regex patterns.
```

Comment:
Though helpful, I'm not sure tldr-pages in the place to store this info.  The more information link is enough IMO&mdash;but I can be overruled.  Thoughts @navarroaxel?
```suggestion
> Tracks the most used directories and enables quickly navigating to them using string or regex patterns.
```

Reply:
What about this?
```suggestion
> Tracks the most used (by frecency) directories and enables quickly navigating to them using string or regex patterns.
```